src/client/html/index.html

```diff
@@ -8,13 +8,95 @@
   <meta name="viewport"
     content="width=device-width, initial-scale=1.0">
   <title>Document</title>
-
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 </head>
 
 <body>
-
-  <div id="root"></div>
-  
+  <header>
+    <nav class="navbar">
+      <div id="logo">
+        로고
+        <!-- TODO webpack 수정 후 이미지 작업 -->
+        <!-- <img src="../assets/image/logo.png" alt="로고"> -->
+      </div>
+      <ul class="navbar-list">
+        <li>면접</li>
+        <li>스터디</li>
+        <li>프로젝트</li>
+        <li>채팅</li>
+        <li class="sign login">로그인</li>
+      </ul>
+    </nav>
+    <div class="searchBar-wrap">
+      <div class="searchBar-input-wrap">
+        <input type="text" placeholder="어떤 스터디를 찾으시나요?" class="searchBar-input">
+      </div>
+    </div>
+  </header>
+  <main class="main-wrap">
```

Comment:
저는 이런식으로 헤더랑 메인이랑은 분리해서 가독성을 좋게 해주기는 하는데 , 이 부분은 취향차이가 있으니 참고만 해주세요
```suggestion
  </header>
  
  <main class="main-wrap">
```

Reply:
확인했습니다! 말씀해주신대로 바꿀게요~~